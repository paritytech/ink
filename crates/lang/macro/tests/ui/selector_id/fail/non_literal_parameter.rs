use ink_lang as ink;

const INPUT: &str = "test";
const _: [u8; 32] = ink::selector_id!(INPUT);

fn main() {}
