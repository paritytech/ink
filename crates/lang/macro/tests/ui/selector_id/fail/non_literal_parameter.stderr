error: expected string or byte string literal as input: expected literal
 --> $DIR/non_literal_parameter.rs:4:39
  |
4 | const _: [u8; 32] = ink::selector_id!(INPUT);
  |                                       ^^^^^
